app/styles/typography.scss

```diff
@@ -34,7 +34,7 @@
   font-weight: 500;
 }
 
-@mixin footnote {
+@mixin p {
```

Comment:
Thoughts on this change? I wanted to create a class for "normal" text that we could easily use on pages (as opposed to components which define their own css).

It can definitely be updated (sizes, spacing etc) at a later date?